extensions/blocks/simple-payments/save.js

```diff
@@ -1,9 +1,60 @@
 /**
- * External dependencies
+ * WordPress dependencies
  */
-import { RawHTML } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { formatPrice } from './utils';
 
 export default function Save( { attributes } ) {
-	const { productId } = attributes;
-	return productId ? <RawHTML>{ `[simple-payment id="${ productId }"]` }</RawHTML> : null;
+	const {
+		content,
+		currency,
+		featuredMediaUrl,
+		featuredMediaTitle,
+		postLink,
+		price,
+		productId,
+		title,
+	} = attributes;
+	if ( ! productId ) {
+		return null;
+	}
+
+	return (
+		<div className={ `jetpack-simple-payments-wrapper jetpack-simple-payments-${ productId }` }>
```

Comment:
Should we bring in `className` here, for folks using custom CSS classes using the dialog at the bottom of the block sidebar in the editor?

Reply:
It's my understanding that [these classes are automatically added by Gutenberg](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/hooks/generated-class-name.js).

However, seems this block does not support custom class names for some reason:

https://github.com/Automattic/jetpack/blob/b8648d7ca8f3d9209f3b6f95fe00a2e760af7876/extensions/blocks/simple-payments/index.js#L161-L162

<img width="273" alt="Screen Shot 2020-09-28 at 15 10 03" src="https://user-images.githubusercontent.com/1233880/94436528-d715fb00-019c-11eb-8bed-9c3e572e97e4.png">